src/Concerns/BatchableTrait.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Maatwebsite\Excel\Concerns;
+
+if (trait_exists('\Illuminate\Bus\Batchable')) {
+    trait BatchableTrait
+    {
+        use \Illuminate\Bus\Batchable;
+
+        public function batchCancelled()
+        {
+            $batch = $this->batch();
+
+            return $batch !== null && method_exists($batch, 'cancelled') && $batch->cancelled();
+        }
+    }
+} else {
+    trait BatchableTrait
+    {
```

Comment:
I think we can add the methods we need else where like batchCancelled here as well, so we don't have to do a method_exist check there. We can return false here

Reply:
Done